mesonbuild/dependencies/dev.py

```diff
@@ -700,6 +702,110 @@ def __init__(self, environment: 'Environment', kwargs: JNISystemDependencyKW):
 packages['jdk'] = JDKSystemDependency
 
 
+class DiaSDKSystemDependency(SystemDependency):
+
+    def _try_path(self, diadir: str, cpu: str) -> bool:
+        if not os.path.isdir(diadir):
+            return False
+
+        include = os.path.join(diadir, 'include')
+        if not os.path.isdir(include):
+            mlog.error('DIA SDK is missing include directory:', include)
+            return False
+
+        lib = os.path.join(diadir, 'lib', cpu, 'diaguids.lib')
+        if not os.path.exists(lib):
+            mlog.error('DIA SDK is missing library:', lib)
+            return False
+
+        bindir = os.path.join(diadir, 'bin', cpu)
+        if not os.path.exists(bindir):
+            mlog.error(f'Directory {bindir} not found')
+            return False
+
+        found = glob.glob(os.path.join(bindir, 'msdia*.dll'))
+        if not found:
+            mlog.error("Can't find msdia*.dll in " + bindir)
```

Comment:
```suggestion
            mlog.error('Can't find msdia*.dll in ' + bindir)
```

Reply:
You probably missed that there's a single quote inside the string. Should I prefer `'Can\'t find msdia*.dll in '` ?

Reply:
Oups! Yes I missed it. You can keep the double quote then.